coops-ui/src/App.vue

```diff
@@ -226,6 +228,9 @@ export default {
     storeEntityInfo (response) {
       if (response && response.data && response.data.business) {
         this.setEntityName(response.data.business.legalName)
+        // TODO: Replace placeholder with `response.data.business.legalType` when legalTypes are seeded.
```

Comment:
Severin: Production code (ie, code committed to master branch) should not contains TODOs (although IMHO they can contain 'FUTURE' comments for work to be done in a future task).

Cameron: Definitely agree. A lot of work going forwards depends on legalTypes having values. Until that value is seeded in production i'm not sure how else to move forward, when the only repo we have is to PR into bcgov/master. Should we wait until that is done before moving forward?

Severin: As Katie mentioned in RC, perhaps this code change should be in a separate branch to keep all functionality together, and then merged into master as a whole? We can discuss as a team if helpful.

Reply:
It was agreed to implement this code, but now this PR is dependent on https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/1370.